examples/event-driven-price-prediction/transform.py

```diff
@@ -0,0 +1,93 @@
+import pandas as pd
+from collections import defaultdict
+
+# In-memory storage to hold the latest events from each service
+event_cache = defaultdict(dict)
```

Comment:
We don't provide such functionality yet.

It could be only in a lucky case, when a pod with a function has not been restarted. But at least, after ~3-10 mins a pod is terminated and will be recreated with a new event.

Reply:
Yes, very good point! I am already thinking of adding external storage to simulate the in-memory storage if small stateful transformation.